tox.ini

```diff
@@ -1,13 +1,15 @@
 [tox]
-envlist = py27,py33,py34,pypy,docs,pep8,py3pep8,gendoc
+envlist = {py27,py33,py34,pypy},docs,pep8,py3pep8,gendoc,coverage-report
```

Comment:
What does adding the `{...}` change? (Honest question, I am trying to figure travis/tox-env speak)